upload.js

```diff
@@ -0,0 +1,28 @@
+var execSync = require('child_process').execSync
+var spawn = require('win-spawn')
```

Comment:
good catch! seems they didn't publish the readme with the deprecation warning. will replace with https://github.com/IndigoUnited/node-cross-spawn